src/main/java/org/openstreetmap/atlas/geography/atlas/builder/RelationBean.java

```diff
@@ -202,6 +211,26 @@ public boolean equals(final Object other)
         return false;
     }
 
+    /**
+     * Check if the two beans are the same, without looking at the bean order. Also, ensure that
+     * their explicitlyExcluded sets match.
+     *
+     * @param other
+     *            The other object
+     * @return True if the other object satisfies {@link RelationBean#equals(Object)} AND has a
+     *         matching explicitlyExcluded set.
+     */
+    public boolean equalsIncludingExplicitlyExcluded(final Object other)
+    {
+        if (other instanceof RelationBean)
```

Comment:
Instead of creating local variables how about
```
return (other instanceof RelationBean) && this.equals(other)
                && this.explicitlyExcluded.equals(((RelationBean) other).explicitlyExcluded);
```